docs/api/effector/Effect.md

```diff
@@ -57,3 +111,54 @@ The only requirement for function:
 <hr>
 
 ## Effect Properties
+
+### <a id='done'></a>[`.done`](#done)
+
+_Event_ triggered when promise from _thunk_ is *resolved*
+
+#### Arguments
+
+Event triggered with object of `params` and `result`:
+
+(_`params`_): Arguments passed to effect call
+(_`result`_): Result of resolved promise
+
+#### Example
+
+```js
+const effect = createEffect()
+
+effect.use((value) => Promise.resolve(value + 1))
+
+effect.done.watch(({ params, result }) => {
+  console.log("Done with params", params, "and result", result)
+})
+
+effect(2) // >> Done with params 2 and result 3
+```
+
+
+### <a id='fail'></a>[`.fail`](#fail)
+
+_Event_ triggered when promise from _thunk_ is *rejected* or thunk throws.
+
+#### Arguments
+
+Event triggered with object of `params` and `error`:
+
+(_`params`_): Arguments passed to effect call
+(_`error`_): Error catched from thunk
+
+#### Example
+
+```js
+const effect = createEffect()
+
+effect.use((value) => Promise.reject(value - 1))
+
+effect.fail.watch(({ params, result }) => {
```

Comment:
I'm not sure but `.fail` has `error` instead of `result`

Reply:
yes